app/states/fantasyState.js

```diff
@@ -57,8 +57,8 @@ const fantasyState = {
     itemGraveyard = []
 
     playerObject.movePlayer()
-    itemCollision(playerObject, itemGraveyard)
-    enemyCollision(playerObject, graveyard)
+    itemCollision(playerObject, localState.loot)
```

Comment:
@gabrielwr I fixed this in previous PR, don't need localState.loot , need itemGraveyard in second arg